src/pages/[teamName]/[projectName]/configuration.tsx

```diff
@@ -474,6 +479,14 @@ const ConfigurationPage = withError<
             overflow="auto"
           >
             <Card heading="Build settings" id={`build-settings`}>
+              <Alert status="warning" alignItems="flex-start">
```

Comment:
Since this alert is currently hitting the divider below the card title, I'd suggest one of two fixes:
1. Move this out of the card, above all of the cards
2. Add `mt={4}`

Reply:
I'm torn... I feel like option 1 looks better but option 2 makes more sense since the alert is related directly to the build inputs... 🤔 Wdyt @KenzoBenzo?

_Option 1_
<img width="603" alt="Screen Shot 2020-06-09 at 2 46 40 PM" src="https://user-images.githubusercontent.com/26869552/84149083-2c989180-aa60-11ea-9f58-fd0a12dc2266.png">

_Option 2_
<img width="578" alt="Screen Shot 2020-06-09 at 2 46 10 PM" src="https://user-images.githubusercontent.com/26869552/84149105-37ebbd00-aa60-11ea-9a08-fc0c6dcd1242.png">

Reply:
I'd agree on option 1! I would believe we will have more configuration values that will have the same _rule_

Reply:
Ok makes sense and thanks! I'll update 🚀